Sources/SwiftFormatPrettyPrint/TokenStreamCreator.swift

```diff
@@ -2606,6 +2606,24 @@ fileprivate final class TokenStreamCreator: SyntaxVisitor {
       tokens: .break(.close(mustBreak: forcesBreakBeforeRightParen), size: 0), .close)
   }
 
+  /// Applies formatting to a collection of enum case parameters for a decl.
+  ///
+  /// - Parameters:
+  ///    - parameters: A node that contains the parameters that can be passed to a decl when its
+  ///      called.
+  ///    - forcesBreakBeforeRightParen: Whether a break should be required before the right paren
+  ///      when the right paren is on a different line than the corresponding left paren.
+  private func arrangeEnumCaseParameterClause(
```

Comment:
I just duplicated this method for `EnumCaseParameterClauseSyntax`. Alternatives would be
1. Make this function take a `leftParen: TokenSyntax, isEmpty: Bool, rightParen: TokenSyntax` so the logic can be shared with `arrangeParameterClause`
2. Create a protocol that offers the three parameters described in (1) and make both `ParameterClauseSyntax` and `EnumCaseParameterClauseSyntax` conform to that

(2) seemed like quite an overkill to me and (1) seemed ugly as well. Let me know if you’ve got preferences @allevato.

Reply:
This is fine; the function is small enough that duplicating it is probably better than overengineering a generic thing that would only be used in one extra place.

Reply:
That’s what I thought as well. But good to hear you’re on the same page.